inc/console/migration/domainsplugintocorecommand.class.php

```diff
@@ -587,7 +587,7 @@ protected function importDomainItems() {
             if (isset($core_items[$domains_id.$itm['itemtype'].$itm['items_id']])) {
                $core_item = $core_items[$domains_id.$itm['itemtype'].$itm['items_id']];
                $message = sprintf(
-                  __('Updating existing domain item %s...'),
+                  __('Skip existing domain item %s...'),
                   $domains_id . ' ' . $itm['itemtype'] . ' ' . $itm['items_id']
                );
             } else {
```

Comment:
Adding a `continue` here may help to clean the code on following lines to make it clearer.
```suggestion
               continue;
            } else {
```

Reply:
With this change, message will not be displayed

Reply:
See my last commit